src/parsers/parseArrayToStringWithAticle.ts

```diff
@@ -0,0 +1,17 @@
+const parseArrayToStringWithArticle = (arr: []): String => {
+  let str = "";
+  let arrSize = arr.length;
+
+  arr.map((item, i) => {
```

Comment:
This is quite imperative.

Maybe handle every but last item with `.join(', ')` and just concat ' e ' + last item would be better.

I found an old function is my gists. Is is ES5, but will help you.

```js
function toList (items) {
  if (items.length === 0) {
    return "";
  } else if (items.length === 1) {
    return String(items[0]);
  } else {
    const last = items.length - 1;
    return items.slice(0, last).join(", ") + " e " + items[last];
  }
}
```